spec/features/work_packages/new_work_package_spec.rb

```diff
@@ -55,8 +51,34 @@ def save_work_package!(expect_success=true)
     end
   end
 
+  def create_work_package(type)
+    work_packages_page.click_toolbar_button 'Work packages'
+    sleep(0.1)
```

Comment:
I tried to remove this `sleep` as it should not be there. Removing it fails specs although there is no obvious reason.

The sequence is
* click create button
* wait
* click "task"
* check if subject empty

removing the `wait` step makes `check if subject empty` fail.